src/converter.ts

```diff
@@ -59,6 +60,7 @@ module powerbi.extensibility.visual {
 
             if (dataViewCategorical.categories && dataViewCategorical.categories.length > 0) {
                 this.category = dataViewCategorical.categories[0];
+                this.image = dataViewCategorical.categories[1]; // may be undefined
```

Comment:
Are you sure that image category is always in the first index of array?
Can we specify image before the main category?